geoportailv3/models.py

```diff
@@ -17,6 +17,10 @@
 Base = sqlahelper.get_base()
 
 
+LayerWMTS.__acl__ = [DENY_ALL]
+WMTSDimension.__acl__ = [DENY_ALL]
```

Comment:
Ok, so the changelog indicates that it should be done for `LayerWMTS` and `WMTSDimension`, but you're saying that we should not do it in the case of the geoportailv3 project. I don't understand why.